tests/30rooms/03members-remote.pl

```diff
@@ -255,15 +255,15 @@
    check => sub {
       my ( $creator, $remote_user, $room_alias_name ) = @_;
 
-      matrix_create_room( $creator,
+      matrix_create_room_synced( $creator,
          room_alias_name  => $room_alias_name,
          creation_content => {
             "m.federate" => JSON::false,
          },
       )->then( sub {
          my ( undef, $room_alias ) = @_;
 
-         matrix_join_room( $remote_user, $room_alias )
+         matrix_join_room_synced( $remote_user, $room_alias )
             ->main::expect_http_403;
```

Comment:
these things (there are quite a lot of them) which check for a 403 response should probably not be synced. Obviously if we're expecting the operation to fail, then there is no point polling `/sync`, but moreover: we want to be sure it is the `/join` that is returning 403, not the `/sync`.

Reply:
Good point, didn't think about that. Restored the normal version where it makes sense.